kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -17,6 +17,7 @@
 
     deployIstio: false,
 
+    exposeRestApi:: false,
```

Comment:
The default should probably be true.

Reply:
https://github.com/tensorflow/serving/blob/9e960b37607da80b8c4c06754e0873c71ae1cd2f/tensorflow_serving/model_servers/main.cc#L334

The http server is only started if it's set, so I think I should only set it when needed.

Reply:
I'm suggesting that by default we should deploy TFServing with the REST API enabled and people can disable it if they want to.

Reply:
My reason for keeping it an opt-in variable was because the older versions don't have the REST API feature.

Alternatively, we could update the default images in the lib file to use TF 1.8. I suspect anyone would want to use a lower version anyways now that 1.10 is out so this should be a safe choice.

Reply:
I think that makes sense.